### PR TITLE
Unify template use and upgrade operations

### PR DESCRIPTION
**Pull request by michaljurecko**
Part of: https://keboola.atlassian.net/browse/PSGO-45

Changes:
- Template use and upgrade operations share common code.

### CODE REVIEW

File: test/cli/template-rename/ok/out/.keboola/manifest.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
...
+        "KBC.KAC.templates.instances": "[{\"instanceId\":\"inst456\",\"instanceName\":\"My Instance\",\"templateId\":\"my-template-id\",\"repositoryName\":\"keboola\",\"version\":\"1.0.0\",\"created\":{\"date\":\"2022-01-01T01:00:00Z\",\"tokenId\":\"123\"},\"updated\":{\"date\":\"2022-01-01T01:00:00Z\",\"tokenId\":\"123\"},\"mainConfig\":{\"configId\":\"5678\",\"componentId\":\"keboola.orchestrator\"}},{\"instanceId\":\"inst123\",\"instanceName\":\"New Name\",\"templateId\":\"my-template-id\",\"repositoryName\":\"keboola\",\"version\":\"1.0.0\",\"created\":{\"date\":\"2022-01-01T01:00:00Z\",\"tokenId\":\"123\"},\"updated\":{\"date\":\"%s\",\"tokenId\":\"%s\"},\"mainConfig\":{\"configId\":\"1234\",\"componentId\":\"keboola.orchestrator\"}}]"
```
Comment (michaljurecko) [inline test/cli/template-rename/ok/out/.keboola/manifest.json:37]: Zmena poradia, ... v tomto teste su akorat `instanceID` nastavene na pekne hodnoty `inst123`, `inst456`, ale realne su to nahodne hodnoty.

File: test/cli/template-test-create/create/expected-stdout [hunk 2]
```diff
@@ -1,4 +1,4 @@
-New objects from "keboola/my-template/0.0.1" template:
+Objects from "keboola/my-template/0.0.1" template:
   + C main/extractor/ex-generic-v2/empty-instance
```
Comment (michaljurecko) [inline test/cli/template-test-create/create/expected-stdout:2]: Zjednotene hlasky medzi use/upgrade operaciou.

Comment (michaljurecko) [reply test/cli/template-test-create/create/expected-stdout:2]: Ine zmeny v testoch nie su.

**Review by michaljurecko — COMMENTED** [review]
Template `use` a `upgrade` operacie mali +- nakopirovany rovnaky kod.
Aby som fix pre shared codes nemusel pridavat tiez 2x, tak v tomto PR najprv odstranujem duplicitny kod.

**Review by JakubMatejka — APPROVED** [review]

**Pull request merged**